common/config/config.go

```diff
@@ -68,6 +68,11 @@ type Config struct {
 	Inference struct {
 		ServerAddr string `envconfig:"STARHUB_SERVER_INFERENCE_SERVER_ADDR" default:"http://localhost:8000"`
 	}
+
+	Space struct {
+		// base url for space api running in k8s cluster
+		K8sEndpoint string `envconfig:"STARHUB_SERVER_SPACE_K8sEndpoint" default:"http://localhost:8080"`
```

Comment:
`STARHUB_SERVER_SPACE_K8sEndpoint` should be `STARHUB_SERVER_SPACE_K8S_ENDPOINT`